apps/web/src/server/views/validation/appealOutcomeInvalidReason.njk

```diff
@@ -0,0 +1,146 @@
+{% extends "_layouts/layout-base.njk" %}
```

Comment:
Can you please move the contents of this file in `invalid-appeal-outcome.njk`. Thank you.

Reply:
Also the templates have been simplified so the `{% block header %}` for example is not needed anymore. Just put the content of the page and skip the rest.